pkgs/development/compilers/openjdk/darwin/default.nix

```diff
@@ -50,6 +50,7 @@ let
 
     passthru = {
       home = jdk;
+      inherit gtk3;
```

Comment:
Oh and please add a comment here explaining the situation, and indicating that gtk3 is not an input of this package.